client/src/components/frequencyTag/index.js

```diff
@@ -0,0 +1,72 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+
+const Box = styled.div`
+  display: flex;
+  background-color: ${(props) => props.theme.colors.fullWhite};
+  height: 3.5rem;
+  width: 20rem;
+  padding: 0 2rem;
+  border-radius: 7px;
+  box-shadow: 2px 10px 30px rgba(0, 0, 0, 0.05);
+  align-items: center;
+
+  @media screen and(max-width: 769px) {
+      width: 85%;
+  }
+`;
+
+const Text = styled.p`
+  color: ${(props) => props.theme.colors.checkboxGray};
+  font-weight: 500;
+`;
+
+const Tag = styled.p`
+  color: #42B7CB;
+  border: 0.15rem solid #42B7CB;
+  border-radius: 1rem;
+  padding: 0.2rem 1rem;
+  margin: 0 1rem;
+  text-align: bottom;
+  font-weight: 500;
+
+  &.once {
+    background: rgba(66, 183, 203, 0.08);
+  }
+  
+  &.twice {
+    background: rgba(66, 183, 203, 0.15);
+  }
+
+  &.three-times {
+    background: rgba(66, 183, 203, 0.25);
+  }
+
+  &.greater-three {
+    background: rgba(66, 183, 203, 0.35);
+  }
+`;
+
+export const FrequencyTag = ({ frequency, weekly }) => {
+    const Rate = weekly ? <><Text>per week</Text></> : <><Text>per month</Text></>;
+    
+    return (
+        <Box>
+            <Text>Meets</Text>
+            {(() => {
+                if (frequency==1) {
```

Comment:
this technically works because you are using the == operator (which allows type conversion equality) instead of the === operator, but it is best practice to avoid using this to avoid some weird bugs that can occur.
aka rn you have
`"1" == 1` but you should have `"1"==="1"`